apis/Google.Cloud.Translation.V2/Google.Cloud.Translation.V2/TranslationClientImpl.cs

```diff
@@ -132,12 +132,19 @@ public override IList<TranslationResult> TranslateHtml(IEnumerable<string> htmlI
         }
 
         /// <inheritdoc />
-        public override IList<Detection> DetectLanguage(string text)
+        public override Detection DetectLanguage(string text) =>
```

Comment:
I considered that... the async version is sufficiently implementation-laden that I thought it might be better not to. Happy to change it though.